test/e2e/mixedcpus_test.go

```diff
@@ -96,17 +78,130 @@ var _ = Describe("Mixedcpus", func() {
 		})
 
 		It("can have more than one pod accessing shared cpus", func() {
+			dp2 := createTestingDeployment("dp-test2")
+			items, err := pods.OwnedByDeployment(fixture.Ctx, fixture.Cli, dp2)
+			Expect(err).ToNot(HaveOccurred())
+			replicas := int(*dp2.Spec.Replicas)
+			Expect(len(items)).To(Equal(replicas), "expected to find %d pods for deployment=%q found %d; %v", replicas, dp2.Name, len(items), items)
+			pod2 := &items[0]
+
+			By("check the second pod successfully deployed with shared cpus")
+			cpus, err := pods.GetAllowedCPUs(fixture.K8SCli, pod2)
+			Expect(err).ToNot(HaveOccurred())
+
+			sharedCpus := GetSharedCPUs()
+			Expect(sharedCpus).ToNot(BeEmpty())
+			sharedCpusSet := e2ecpuset.MustParse(sharedCpus)
 
+			By(fmt.Sprintf("checking if shared CPUs ids %s are presented under pod %s/%s", sharedCpus, pod2.Namespace, pod2.Name))
+			intersect := cpus.Intersection(sharedCpusSet)
+			Expect(intersect.Equals(sharedCpusSet)).To(BeTrue(), "shared cpu ids: %s, are not presented. pod: %v cpu ids are: %s", sharedCpusSet.String(), fmt.Sprintf("%s/%s", pod2.Namespace, pod2.Name), cpus.String())
 		})
 
 		It("should contain OPENSHIFT_MUTUAL_CPUS environment variable", func() {
+			sharedCpus := GetSharedCPUs()
+			Expect(sharedCpus).ToNot(BeEmpty())
+
+			sharedCpusSet := e2ecpuset.MustParse(sharedCpus)
+			out, err := pods.Exec(fixture.K8SCli, pod, []string{"/bin/sh", "-c", "printenv | grep OPENSHIFT_MUTUAL_CPUS"})
```

Comment:
maybe just use `printenv OPENSHIFT_MUTUAL_CPUS`

Reply:
Sure. What about
```
out, err := pods.Exec(fixture.K8SCli, pod, []string{"/bin/sh", "-c", "printenv OPENSHIFT_MUTUAL_CPUS"})
```
?
or even
```
out, err := pods.Exec(fixture.K8SCli, pod, []string{"/bin/printenv",  "OPENSHIFT_MUTUAL_CPUS"})

Reply:
awesome, I didn't know it's possible to print specific value. so good to have such useful reviews.